pkg/constants/constants.go

```diff
@@ -153,6 +153,14 @@ const (
 
 	// OSMMeshConfig is the name of the OSM MeshConfig
 	OSMMeshConfig = "osm-mesh-config"
+
+	// EnvoyActiveHealthCheckPath is the HTTP endpoint to be used to receive
+	// active health checks.
+	EnvoyActiveHealthCheckPath = "/healthz/osm"
```

Comment:
Which application implements `/healthz/osm`?

Reply:
That endpoint is handled by the health check HTTP filter, so Envoy responds to that all by itself.